### ISSUE TITLE
Make train setup

### ISSUE THREAD

**Issue by icebreaker2** [icebreaker2/AI#40]
The train setup should:
* train with lower batch size first (Make small mini batches approximating stochastic gradient descent) to find good spots of convergence by jumping more likely from minima to minima
* save model weights
* train with higher batch size (Make greater mini batches (e. g. double the amount)) to better converge to found minimas
* Optionally: Make some epochs on whole dataset to finally increase convergence by a few points)
